### DISCUSSION TITLE
Masterscope and Common Lisp

### DISCUSSION THREAD

**Discussion by pamoroso** [Interlisp/medley#1792] [Q&A]
According to the [Medley 1.0 release notes](http://bitsavers.org/pdf/xerox/interlisp-d/198809_Medley_1.0/400006_Lisp_Release_Notes_Medley_Release_1.0_Sep88.pdf) (page 21 of the PDF):

> MASTERSCOPE contains Common Lisp query support allowing you to ask questions about Common Lisp code that could previously be asked only of Interlisp-D code. Currently, questions specific to Common Lisp constructs are not supported.

This is a bit confusing. Can Masterscope answer question on Common Lisp code or not? What questions can it answer and not answer about Common Lisp code?

My understanding is Masterscope can answer questions on constructs common to both dialects, such as bindings and calls, but not on specific Comon Lisp features like structures, conditions, and CLOS.

**Comment by pamoroso** [comment]
The [Masterscope manual](https://files.interlisp.org/medley/library/MASTERSCOPE.TEDIT.pdf) also covers the types of Common Lisp queries the tool supports.

**Reply by masinter** [reply]
Should e close this issue?

**Reply by pamoroso** [reply]
It can be close but isn't this a discussion rather than an issue?
